### PR TITLE
add help option

### PR DESCRIPTION
**Pull request by azakordonets**
* Added help message in a separate file ( copied mostly from README)
* Added test to it

### CODE REVIEW

**Comment by mikefrancis** [comment]
Thanks for this!

What you've done is great however I think it's maybe a little bit more complex than I envisaged it.

When you type `/moo help` I was just hoping you could get a couple of lines back with the available commands rather than examples too.

Can you please amend?

**Comment by azakordonets** [comment]
Done.

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require('dotenv').config({ silent: true });
+var fs = require('fs');
```
Comment (mikefrancis) [inline src/index.js]: This isn't needed any more.

File: test/index.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ process.env.SLACK_TOKEN = 'someToken';
 
 var app = require('./../src/index');
 var request = require('supertest')(app);
+var fs = require('fs');
+var path = require('path');
```
Comment (mikefrancis) [inline test/index.js]: Neither are these.

Comment (azakordonets) [inline test/index.js]: Sorry about that - pushed changes in a hurry. Fixed now

File: README.md [hunk 3]
```diff
@@ -58,6 +58,13 @@ Gives :
                 ||     ||
 ```
 
+You can also look for a help, if you forgot options format :
```
Comment (mikefrancis) [inline README.md]: Change this line to:

```
Type the following to view usage options:
```

File: README.md [hunk 4]
```diff
@@ -58,6 +58,13 @@ Gives :
                 ||     ||
 ```
 
+You can also look for a help, if you forgot options format : 
+```bash
+/moo help
+```
+
+Will give you a nice help message with examples of can you use `/moo` command
```
Comment (mikefrancis) [inline README.md]: Remove this line.

File: README.md [hunk 5]
```diff
@@ -58,6 +58,13 @@ Gives :
                 ||     ||
 ```
 
+Type the following to view usage options::
```
Comment (mikefrancis) [inline README.md]: You have an extra `:`

File: README.md (hunk 4 above)
Comment (mikefrancis) [inline README.md]: Still need to remove this line:
```
Will give you a nice help message with examples of can you use `/moo` command
```

File: README.md (hunk 5 above)
Comment (azakordonets) [inline README.md]: I need to work more on my attention :)

**Comment by mikefrancis** [comment]
Awesome, merged! Thanks a lot.

**Pull request merged**